flooddrake/flux.py

```diff
@@ -0,0 +1,244 @@
+
+from __future__ import division # Get proper divison
+
+import math
+import random
+import numpy as np 
+
+
+from firedrake import *
+parameters["reorder_meshes"] = False
+
+
+class Fluxes():
+    
+    """ Defines the interior and boundary fluxes for one or two dimensional SWEs solved using Firedrake DG model
+    
+    """ 
+    
+    
+    def __init__(self,N,V):
+        
+        """
+        
+        	:param N: Normal
+        	:type N: :class:`FacetNormal'
+        	
+        	:param V: Function Space of the state vector
+        	:type V: :class:`MixedFunctionSpace'
+        	
+        
+        """
+        
+        # set state functions and function spaces
+        self.d=V[0].mesh().geometric_dimension()
+        self.N=N
+        self.g=9.81
```

Comment:
Why not have a parameters class?  E.g. like in https://github.com/firedrakeproject/gusto/blob/master/dcore/configuration.py